pkg/kubelet/container/runtime.go

```diff
@@ -119,6 +119,8 @@ type Runtime interface {
 	// stream the log. Set 'follow' to false and specify the number of lines (e.g.
 	// "100" or "all") to tail the log.
 	GetContainerLogs(pod *api.Pod, containerID ContainerID, logOptions *api.PodLogOptions, stdout, stderr io.Writer) (err error)
+	// Remove a container. If the container is still running, an error is returned.
+	RemoveContainer(containerID ContainerID) error
```

Comment:
Renamed it to DeleteContainer to fit the semantics of the new runtime interface.